source/release-notes/1.8.txt

```diff
@@ -21,7 +21,18 @@ Release Notes for mongosync 1.8
 
 **Upcoming**
 
-{+c2c-product-name+} Beta Program 
+Updates to ``mongosync`` ``/progress`` API endpoint
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Returns a top-level boolean ``success`` field.
+
+- Returns a new ``totalEventsApplied`` field.
+
+- Reports per-partition progress tracking
+
+See the :ref:`c2c-api-progress` documentation for more details.
+
+{+c2c-full-beta-program+}
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. include:: /includes/beta-program-intro.rst
```

Comment:
With the beta program intro being moved down, I think it should explain that it includes all of the changes for the general binary, and the additional changes listed in the section.

It may also be helpful to have the other release notes be called "General release notes" and the beta ones be called "Beta release notes"

Reply:
Gotcha, updating!

Reply:
I'm not seeing a sentence explaining that the beta includes changes from the general binary after: "Starting in version 1.8.0, Cluster-to-Cluster Sync introduces the Cluster-to-Cluster Sync Beta Program. With mongosync beta, you can use experimental features with direct support and assistance from MongoDB." I think it would be good to add

Reply:
Ah sorry, I misread + misunderstood the original comment — thought it referred to just the movement & title making it clearer. Updating with the sentence.